wallet/src/de/schildbach/wallet/request/CreateAdReq.java

```diff
@@ -0,0 +1,36 @@
+package de.schildbach.wallet.request;
+
+public class CreateAdReq {
+
+    /**
+     * phone : 19033949493
+     * email : demo@geni.to
```

Comment:
remove all instances of "geni.to" from this project. Replace with anything else!

Reply:
@rgenito It's updated with other test domain.